lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -41,26 +44,39 @@ class UserProfileBloc extends Cubit<UserProfileState> with HydratedMixin {
       );
     }
     emit(profile);
-    registerForNotifications();
+    _breezLib.nodeStateStream.firstWhere((nodeState) => nodeState != null).then((_) {
+      registerForNotifications();
+    });
   }
 
   Future registerForNotifications() async {
     _log.info("registerForNotifications");
     String? token = await _notifications.getToken();
     if (token != null) {
-      if (token != state.profileSettings.token) {
-        _log.info("Got a new token, registering…");
-        var userID = await _breezServer.registerDevice(token, token);
-        emit(state.copyWith(
-          profileSettings: state.profileSettings.copyWith(
-            token: token,
-            userID: userID,
-            registrationRequested: true,
-          ),
-        ));
-      } else {
-        _log.info("Token is the same as before");
-      }
+      _log.info("Got a new token, registering…");
```

Comment:
```suggestion
      _log.info("Retrieved token, registering…");
```

It's not necessarily a new token.